Backend/TimeTracker.Test/ProjectTests.cs

```diff
@@ -143,6 +147,74 @@ await projectController.UpdateProjectDetails(new ProjectDetailsDTO()
 
             Assert.IsTrue(userProjects.Data.Count == 1);
             Assert.IsTrue(userProjects.Data[0].Id == project.Data.Id);
+        }
+
+        [TestMethod]
+        public async Task RemoveUserFromProject()
+        {
+            var createResult = await projectController.CreateProject(new ProjectCreateDTO()
+            {
+                ClientName = "Test Client",
+                Description = "Test Description",
+                ProjectName = "Test Name",
+                Tags = new List<string>()
+                {
+                    "Test Tag 1",
+                    "Test Tag 2"
+                }
+            });
+
+            Assert.IsTrue(createResult.Success);
+
+            var project = await projectController.GetProjectById(createResult.Data);
+
+            Assert.IsTrue(project.Data.Description == "Test Description");
+
+            await projectController.UpdateProjectDetails(new ProjectDetailsDTO()
+            {
+                Description = "Updated Description",
+                ProjectId = createResult.Data
+            });
+
+            var projectUpdated = await projectController.GetProjectById(createResult.Data);
+
+            Assert.IsTrue(projectUpdated.Data.Description == "Updated Description");

```

Comment:
I think all this can be skipped you don't really need to test that the project can be updated, this is done in a different test